package.json

```diff
@@ -6,12 +6,15 @@
   "author": "EvitaDAO team",
   "license": "MIT",
   "dependencies": {
-    "@nomiclabs/hardhat-ethers": "^2.0.6",
     "@openzeppelin/contracts-upgradeable": "^4.6.0",
     "ethers": "^5.6.8",
     "hardhat": "^2.9.9"
   },
   "devDependencies": {
+    "@openzeppelin/hardhat-upgrades": "^1.19.0",
+    "@nomiclabs/hardhat-ethers": "^2.0.6",
+    "@nomiclabs/hardhat-truffle5": "^2.0.6",
+    "@nomiclabs/hardhat-web3": "^2.0.0",
```

Comment:
Pls don't mix frameworks. We already use hardhat/ethers. Pls use them instead of truffle/web3js

Reply:
I've tried not to add it: 
```
Plugin @nomiclabs/hardhat-truffle5 requires the following dependencies to be installed: @nomiclabs/hardhat-web3.
```